### PR TITLE
Adds Amber Task Force. CentCom, ERT and Inquisition equipment Revamp

### PR DESCRIPTION
**Pull request by Hopekz**
**TL;DR:**
Revamped Blue and Red ERT equipment load-outs.
Revamped CentCom Commander and CentCom Offical equipment load-outs.

Update 2:
Revamped the Inquisition response team equipment load-outs.

Update 3:
Recreated the Amber ERT into the Amber Task Force. This new team was built from the ground up.

CentCom
------

**CentCom official:**
- Replaced satchel with a leather satchel
- Added internals box to backpack
- Now mindshielded
- replaced energy gun with a telescopic baton
- moved clipboard from hand to backpack
- **Adds a mindshield to the centcom costume syndicate box for nukies now that real officials are mindshielded. (Pointed out by Gamer01)**

**CentCom Commander:**
- Commander gets a security belt
- Moved revolver to suit storage from the belt slot
- Added an internals box to backpack
- Now mindshielded
- (still not immune to flash or sound because of basic headset + normal eyepatch)

ERT blue alert::
----------

**Blue ERT Commander:**
- security gas mask moved from backpack to face
- energy gun moved to suit storage instead inside the backpack 
- Keeps his duplicate stun baton because he's the commander. So can "give" it to someone.

**Blue Security ERT:**
- tactical energy gun replaced with an energy gun
- energy gun moved to suit storage instead of the backpack
- security gas mask moved from backpack to face
- Removed stun baton from the backpack. They already get one in the belt and this is blue alert
- Box of handcuffs replaced with a box of zipties

**Blue Engineer ERT:**
- security gas mask moved from backpack to face
- removed energy gun; keeps baton
- replaced stun baton with the telescopic baton
- Removed medkit in hands
- added extra compressed matter cartridge to free pocket
- moved extended-capacity emergency oxygen tank to suit slot
- removed internals box now that mask is on the face and o2 tank is on suit slot
- added Epinephrine medipen to backpack where the box used to be

**Blue Medic ERT:**
- security gas mask moved from backpack to face
- removed energy gun
- replaced stun baton with the telescopic baton
- removed belt
- moved telescopic baton to belt slot
- moved the medical beamgun from backpack to suit storage
- moved combat stimulant injector from backpack to pocket
- removed regular medkit held in hands
- Added toxin first aid kit to backpack
- Added fire first aid kit to backpack
- Added brute first aid kit to backpack
- Moved extended-capacity emergency oxygen tank from internals box to right pocket
- Removed internal box now that mask is on face and tank is in pocket

**Blue Janitor ERT:**
- security gas mask moved from backpack to face
- removed stun baton from the backpack. Their suits cannot hold these.
- moved trash bag of holding from hand to suit storage slot

ERT red alert::
----------

**Red ERT Commander:**
- moved SWAT mask from backpack to face
- replaced the pulse pistol with a tactical energy gun
- moved the tactical energy gun to suit storage instead of their backpack
- Keeps his duplicate stun baton because he's the commander. So can "give" it to someone.

**Red ERT Security:**
- moved SWAT mask from backpack to face
- replaced pulse carbine with the tactical energy gun
- moved the tactical energy gun to suit storage instead of their backpack
- Keeps his duplicate stun baton so he can "give" it to someone.

**Red ERT Engineer:**
- moved SWAT mask from backpack to face
- removed pulse pistol
- replaced stun baton with the telescopic baton
- Added box of smart foam grenades
- replaced toolbelt with an engineering tool belt
- moved extended-capacity emergency oxygen tank to suit slot
- removed internals box now that mask is on the face and o2 tank is on suit slot
- added Epinephrine medipen to backpack where the box used to be

**Red Medic ERT:**
- moved SWAT mask from backpack to face
- moved combat stimulant injector from backpack to left pocket
- removed pulse pistol
- replaced stun baton with the telescopic baton
- moved the medical beamgun from backpack to suit storage
- removed basic medkit from hands
- replaced medical belt with combat defibrillator
- Added toxin first aid kit to backpack
- Added fire first aid kit to backpack
- Added brute first aid kit to backpack
- Added oxygen first aid kit to backpack
- Added advanced first aid kit to backpack
- Moved extended-capacity emergency oxygen tank from internals box to right pocket
- Replaced the internal box with a bodybag box

**Red Janitor ERT:**
- security gas mask moved from backpack to face
- moved trash bag of holding from hand to suit storage
- moved janitor chem sprayer from hand to backpack
- replaced stun baton with the telescopic baton

Update 2:
------

Revamped ERT Chaplain and Inquisition response team loadout.

**ERT Chaplain:**
- security gas mask moved from backpack to face
- moved energy gun from backpack to suit storage
- moved nullrod from backpack to left pocket

Inquisition
-----

**Inquisition Commander:**
- security gas mask moved from backpack to face
- moved energy gun from backpack to suit storage
- moved stun baton from belt to backpack
- moved flashbang baton from belt to backpack
- moved flash baton from belt to backpack
- moved pepperspray baton from belt to backpack
- removed belt
- moved possessed chainsaw sword from hand to belt

**Inquisition Chaplain:**
- security gas mask moved from backpack to face
- moved energy gun from backpack to suit storage
- moved holy hand grenade from backpack to right pocket
- moved nullrod from backpack to left pocket

**Inquisition Security:**
- security gas mask moved from backpack to face
- moved tactical energy gun from backpack to suit storage

**Inquisition Medic:**
- security gas mask moved from backpack to face
- moved energy gun from backpack to suit storage
- Removed basic medic kit in hands
- moved combat stimulant injector from backpack to left pocket
- moved holy water autoinjector from backpack to right pocket
- moved Medical Beamgun from backpack to suit storage
- removed belt
- moved energy gun from backpack to belt slot
- Added toxin first aid kit to backpack
- Added fire first aid kit to backpack
- Added brute first aid kit to backpack

Update 3
-----

Revamped the Amber ERT from the ground up.
**It is completely recreated as the "Amber Task Force" (ATF)**

This is a brand new force able to be sent by Nanotrasen and includes unique items that they carry built specifically for this force.

ghostpoll
group

**Q:** What is the Amber Task Force?
**A:**  The ATS is a Nanotrasen force similar to "space marines". 
They are sent out to terrible situations (After aliens took over etc) and provide a fair and challenging fight to both the antagonists and the force. 
The task force is only armed with lethal weaponry with the exception of the commander which sports a telescopic baton.
**They have obvious weaknesses that can be exploited by their opponents.**

There are 3 different classes in this Loadout
- The Commander
- Soldiers
- Medics

There is 1 Commander and after that, you have a 2/3 chance to get a soldier and a 1/3 chance to roll for a medic until the task force is full.

Similarities between 3 classes
----
- Do not have a backpack. Instead, they carry a modified big tactical oxygen tank on their backs and all the equipment they need in a military vest with different load-outs depending on the class.
- They are not flash immune. The Commander uses the basic sec hud and the Medic uses the basic medical hud
- They are not Atmos immune. They do not come with spacesuits
- They all use the Laser rifle. It is the same gun used in CTF except toned down to be slightly weaker than the usual laser rifle and fire regular lasers.
- All of them have a new modified tactical breath mask. This model is not translucent for anonymity, unlike a regular breathing mask.
- No name on their ID's . This means that if they have their masks up or if the soldier class has his helm down they show up as "Unknown".  Only identified by the Amber Task Force.
- All 3 classes have a combat knife in their combat boots that they can access by alt clicking on the boots.
- All 3 classes have a bulletproof vest and combat gloves for shock protection.

**Main differences in classes:**
-------

Amber Commander
------
commander2
commander
- Has a megaphone because he's the boss
- Has a Telebaton for non-lethal crowd control.
- Sweet cigar!
- Sec hud

Amber Soldier
------
soldier2
soldier
- Most common. For every 3 ATF members, you get 2 Soldiers.
- Riot helm. Meaning that Xenos won't be able to just throw aliens at Soldiers to end the fight
- Really bright flares
- **No more sechud** see "Soldier Update:"

**Soldier Update:**

After careful consideration of the balancing of the 3 different classes I have removed the sechud from the soldier class.
This both makes it a special trait of the commander and gives antags a chance to "trick" soldiers and medics (basically the whole task force except the commander) by not giving the soldiers all the information they need to identify a potential otherwise hidden threat.

This also might encourage the force to stick closer to their Commander as he has the information on who is mind-shielded, wanted or who has the wrong ID access etc.

Amber Medic
------
medic1
medic2
- Has a defib on his back. Instead has an oxygen tank in pocket.
- 3 Survival medipens
- Synthflesh spray (to get corpses back to defib range)
- combat stimulant injector
- only 1 spare magazine for his rifle

#### Changelog

:cl:  Hopek
tweak: Revamped Blue and Red ERT equipment load-outs.
tweak: Revamped CentCom Commander and CentCom Offical equipment load-outs.
rsadd: The Syndicate CentCom official clothing kit now comes with a mindshield now that CentCom officials have mindshields.
tweak: Revamped the Inquisition response team equipment load-outs.
rsadd: Recreated the Amber ERT in to the Amber Strike Force. This new team was built from the ground up.
/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
i'd rather have the centcom official get a mini-egun

**Comment by patpol4** [comment]
small things good things

**Comment by patpol4** [comment]
> i'd rather have the centcom official get a mini-egun

He's a bureaucrat, why would he get a gun at all

**Comment by Hopekz** [comment]
Update:
The nuke ops syndicate Centcom official clothing kit comes with a mindshield now that they are mindshielded.
Can't impersonate Officials without being mindshielded!

**Comment by Marmio64** [comment]
> > i'd rather have the centcom official get a mini-egun
> 
> He's a bureaucrat, why would he get a gun at all

why does hop get a mini-egun then if hes just a bureaucrat

**Comment by patpol4** [comment]
> > > i'd rather have the centcom official get a mini-egun
> > 
> > 
> > He's a bureaucrat, why would he get a gun at all
> 
> why does hop get a mini-egun then if hes just a bureaucrat

HoP is a loot pinata for gamers and second in command so its only reasonable that he gets a ranged weapon

**Comment by yacabo** [comment]
I disagree with disarming the blue alert team of their ranged weapons, they are a team expecting a fight and sometimes they're supposed to be red alert but all the heads are dead. Maybe instead of completely removing their eguns give them a disabler instead?

**Comment by Hopekz** [comment]
> I disagree with disarming the blue alert team of their ranged weapons, they are a team expecting a fight and sometimes they're supposed to be red alert but all the heads are dead. Maybe instead of completely removing their eguns give them a disabler instead?

The goal of this is to make blue the "not expecting a fight but things are bad" team and then the Red transforms in to the "Expecting a fight" team.

**Before these changes:**
Blue: Expecting a fight
Red: I want everyone to die (They get fucking death squad pulse rifles)
Deathsquad: I want everyone to really die in a funny unfair way

**After these changes:**
Blue: I'm not expecting heavy combat but something is wrong and needs fixing
Red: I am expecting a fight and things are really bad
Deathsquad: I want everyone to really die in a funny unfair way

**There is no ramp-up with our ERT's at the moment** it's all "fuck you (blue), fuck you die (red) and you have no choice but to die (death squad)"


As an admin that busses in ERT's I want to be able to send a blue ERT without decimating the antags. 
I want to introduce more ghosts into the round and see everyone put up a fight.

Also; This will result in more blue ERT's being sent out.
Honestly one of the biggest reasons we don't see more of them is that they're just too powerful. 
If I even send a blue team in they'll wipe the floor with antags most of the time.

And I don't even think about sending Red because red is just death squad except with different armor. They literally have the same pulse rifles as the death squad.

**Comment by yacabo** [comment]
I agree with they do need spacing in terms of strength but it still strikes me as odd that they're less equipped than the on board security team when they're the ones that might get sent in when security failed. Then again I might not understand it because I've never been an admin capable of sending them in.

**Comment by Hopekz** [comment]
> I agree with they do need spacing in terms of strength but it still strikes me as odd that they're less equipped than the on board security team when they're the ones that might get sent in when security failed. Then again I might not understand it because I've never been an admin capable of sending them in.

The Blue Security ERT and the Blue Commander ERT are more equipt for taking out threats than a sec officer because they get energy rifles (so even lethal capabilities).
I do not think the medic, janitor and engineer ERT should engage in a combat role especially in a blue ERT.

Blue ERT's are meant to be sent out during blue alert by design but we've never seen them used this way because blue has always been OP to the point where you send them out during "high red" situations. This makes the unchanged Red ERT a deathsquad from the way they're deployed.

Even a few days ago a cult took over the whole station so I sent in a Red ERT forgetting that they're death squads so I had to delete their pulse rifles. I should not have to delete their pulse rifles the red team should handle red threats.

**Comment by yacabo** [comment]
Mh, you do make valid points. I'll assume you're correct in this topic

**Comment by Hopekz** [comment]
Update:

**Revamped ERT Chaplain and the Inquisition response team as well.**

**ERT Chaplain:**
- security gas mask moved from backpack to face
- moved energy gun from backpack to suit storage
- moved nullrod from backpack to left pocket

**Inquisition Commander:**
- security gas mask moved from backpack to face
- moved energy gun from backpack to suit storage
- moved stun baton from belt to backpack
- moved flashbang baton from belt to backpack
- moved flash baton from belt to backpack
- moved pepperspray baton from belt to backpack
- removed belt
- moved possessed chainsaw sword from hand to belt

**Inquisition Chaplain:**
- security gas mask moved from backpack to face
- moved energy gun from backpack to suit storage
- moved holy hand grenade from backpack to right pocket
- moved nullrod from backpack to left pocket

**Inquisition Security:**
- security gas mask moved from backpack to face
- moved tactical energy gun from backpack to suit storage

**Inquisition Medic:**
- security gas mask moved from backpack to face
- moved energy gun from backpack to suit storage
- Removed basic medic kit in hands
- moved combat stimulant injector from backpack to left pocket
- moved holy water autoinjector from backpack to right pocket
- moved Medical Beamgun from backpack to suit storage
- removed belt
- moved energy gun from backpack to belt slot
- Added toxin first aid kit to backpack
- Added fire first aid kit to backpack
- Added brute first aid kit to backpack

**Comment by Moja10** [comment]
I love this PR. This fixes so many issues with the concept of bussing ERTs in. This is a fantastic change for both ERT and antags who have to deal with ERT and don't have to deal with pulse shots now. 

Two things I would recommend:
1. Mini e-gun to Centcomm Official, one problem that I find with bussing these in, if the crew know they are coming, sometimes greytiders and antags will bum rush them knowing they don't have adequate weapons, think like an armed antag going after the CMO who only has a baton, the CMO just capitulates. Mini e-gun is fair and it's what we give to the HoP. IMO I never had a problem with full e-gun but this should be just fine.
2. If it's not too much trouble, please look into Code Amber too.

**Comment by Marmio64** [comment]
i think blue ert should get mini-eguns and inspector still needs one too, since chances are the inspector is checking out what may or may not be a dangerous situation. Maybe even give the inspector a disabler instead or something.

**Comment by Hopekz** [comment]
Update:

Revamped the Amber ERT from the ground up.
**It is completely recreated as the "Amber Task Force" (ATF)**

This is a brand new force able to be sent by Nanotrasen and includes unique items that they carry built specifically for this force.

ghostpoll
group


**Q:** What is the Amber Task Force?
**A:**  The ATS is a Nanotrasen force similar to "space marines". 
They are sent out to terrible situations (After aliens took over etc) and provide a fair and challenging fight to both the antagonists and the force. 
The task force is only armed with lethal weaponry with the exception of the commander which sports a telescopic baton.
**They have obvious weaknesses that can be exploited by their opponents.**

There are 3 different classes in this Loadout
- The Commander
- Soldiers
- Medics

There is 1 Commander and after that, you have a 2/3 chance to get a soldier and a 1/3 chance to roll for a medic until the ERT is full.

Similarities between 3 classes
----
- Do not have a backpack. Instead, they carry a modified big tactical oxygen tank on their backs and all the equipment they need in a military vest with different load-outs depending on the class.
- They are not flash immune. The Commander and Soldier use the basic sec hud and the Medic uses the basic medical hud
- They are not Atmos immune. They do not come with spacesuits
- They all use the Laser rifle. It is the same gun used in CTF except toned down to be slightly weaker than the usual laser rifle and fire regular lasers.
- All of them have a new modified tactical breath mask. This model is not translucent for anonymity, unlike a regular breathing mask.
- No name on their ID's . This means that if they have their masks up or if the soldier class has his helm down they show up as "Unknown".  Only identified by the Amber Task Force.
- All 3 classes have a combat knife in their combat boots that they can access by alt clicking on the boots.
- All 3 classes have a bulletproof vest and combat gloves for shock protection.

**Main differences in classes:**
-------

Amber Commander
------
commander2
commander
- Has a megaphone because he's the boss
- Has a Telebaton for non-lethal crowd control.
- Sweet cigar!
- Sec hud

Amber Soldier
------
soldier2
soldier
- Most common. For every 3 ATF members, you get 2 Soldiers.
- Riot helm. Meaning that Xenos won't be able to just throw aliens at Soldiers to end the fight
- Really bright flares
- **No more sechud** see separate comment below or OP for reasoning.

Amber Medic
------
medic1
medic2
- Has a defib on his back. Instead has an oxygen tank in pocket.
- 3 Survival medipens instead of 1 epipen like the other 2 classes
- Synthflesh spray (to get corpses back to defib range)
- combat stimulant injector
- only 1 spare magazine

**Comment by Hopekz** [comment]
@Moja10 
> with bussing these in, if the crew know they are coming, sometimes greytiders and antags will bum rush them knowing they don't have adequate weapons

Why is this an issue that needs addressing?
I think taking out an official as an antag (if you want to impersonate them or something) is a valid strategy and that kind of play should be encouraged.

Otherwise; After these changes killing an official gives you a telescopic baton and an ID that doesn't have much access to the station itself. 
In my opinion giving the official a mini egun actually makes the described problem worse because he now has a mini egun as "loot" for killing/robbing the official.

Replacing his energy gun with a baton makes him a less worthwhile target.

**Comment by LoliconSlayer** [comment]
The Task Force is cool, might need a renaming as to not trigger too many people who know the IRL ATF. Now I gotta rework the lore to fit these fuckers in

**Comment by Hopekz** [comment]
> The Task Force is cool, might need a renaming as to not trigger too many people who know the IRL ATF. Now I gotta rework the lore to fit these fuckers in

You're the lore man but I picture them as a shock tropper task force from warhammer 40k except from Nanotrasen.
They're all bald and have no facial hair regardless of gender if that helps paint an idea on who they are more.

I want to create a weaker force to send out that isn't a "task force" but a "trooper squad" as well but that is for another PR

**Comment by Hopekz** [comment]
Update:

After careful consideration of the balancing of the 3 different classes I have removed the sechud from the soldier class. 
This both makes it a special trait of the commander and gives antags a chance to "trick" soldiers and medics (basically the whole task force except the commander) by not giving the soldiers all the information they need to identify a potential otherwise hidden threat.

This also might encourage the force to stick closer to their Commander as he has the information on who is mind-shielded, wanted or who has the wrong ID access etc.

**Comment by LoliconSlayer** [comment]
Bald and No Facial hair isn't an issue, it's close enough to standard military dress.
It's mostly this line that I'm in no way committing to the lore as of now:
> The ATS is a Nanotrasen force similar to "space marines"

Just don't code in lore and there won't be too many problems
(I think you meant ATF instead of ATS btw)

Also, still on the topic of their name, maybe rename them to Task Force Amber to avoid any ATF parallels.

**Comment by Marmio64** [comment]
isn't amber like, in between code red and blue? I mean, a code red ERT is more equipped than the amber task force and you say they're gonna be used for fighting xenos, but xenos tend to space areas they're in, so these guys are just going to die. They look cool and all, but I don't see many cases where they'll be allowed to be spawned in AND be useful.

**Comment by Hopekz** [comment]
> isn't amber like, in between code red and blue? They look cool and all, but I don't see many cases where they'll be allowed to be spawned in AND be useful.

I see many cases where I would personally bus these guys in. Gangs for example, on shifts where cults have completely taken over and someone sends a message for centcom to help etc.

Amber was very under used before; most admins that bus in ERT's that I've talked to didn't know what an amber ERT was.
These guys are less equip to deal with most threats and do not have the practical tools or survival methods of a red or sometimes even a blue ERT. 
Even blue ERT's have the added benefits of being atmos immune, flash immune, being able to stun , have tools to fix things etc.
It will be clear as to what situation The Amber Task Force applies for once they are in play.

> I mean, a code red ERT is more equipped than the amber task force and you say they're gonna be used for fighting xenos, but xenos tend to space areas they're in, so these guys are just going to die. 

You are right! 
I want these guys to either put up a fight or be slaughtered. 
These guys do not have tools to deal with their weaknesses if exploited. This is intended. 
I think xenos that space the area in response to these guys should be rewarded by either killing these guys or pinning them down in an area with atmos.

If this task force is countered via their weaknesses they create cool weapon/ammo pickups for whoever is left / the next ERT being sent in (if any). Like all the dead bodies you find in every video game that lets you re-arm!

**Common sense still applies**. 
If there is no atmos in the station to begin with.. don't bus the Amber Task Force in. They can't deal with that. In that case you would send either a blue or a red.

**Comment by alexkar598** [comment]
ATF seems pretty fucking weak with most abilities being one times, synthflesh will deplete very quickly and so will the power packs for the laser rifles(can they be recharged????)

**Comment by Hopekz** [comment]
> ATF seems pretty fucking weak with most abilities being one times, synthflesh will deplete very quickly and so will the power packs for the laser rifles(can they be recharged????)

Yeah, in terms of survivability the force is weaker than the red and blue ERT's we're used to for sure besides the firepower of their rifles. The main advantage is that they can unleash 20 shots per magazine of the laser carbine rounds.

These guys are meant to have a harder time in whatever situation you throw them in to.
They're like glass cannons perse.

The magazines hold laser bullets so they cannot be reloaded.
This is not a down-side in my opinion.
Compared to a laser rifle that the ERT's come with each Soldier has 3x the amount of power of an ERT carrying a laser rifle that cannot be reloaded because they fire 20 shots before needing to recharge their gun fully.

If they fire 3 full magazines out they still have their combat knives in their boots (20 damage a hit)

I think the balance of this force is pretty nice where it is.
I will not be abandoning this project if additional changes are needed as more information is gathered so I'd like to see where they stand right now as is.

**Comment by Alexandria404** [comment]
I dont like the centcom commander to have a security belt tbh

it´s not some valid hunting role and neither some lowly sec officer doing dirty work,  but an RP fun time role and only having the revolver to threaten people is more fitting in my mind.
To only carry a revolver feels so big dickish, important person

Hurts muh shooting and threatening people RP
